Task/update environment setup documentation

Kait211 opened the pull request:
Summary of changes:

* Updated the page120_working_with_jaia_software.md file with necessary revisions and improvements:
    - I created a new section about setting up a new default user if needed.
    -  Under Launching the Simulator in step 5. Launch the simulator in a separate terminal, I added commands that could be needed if  default user does not have correct permission to launch.
    - Under Launching the Simulator in step 6. Troubleshooting the simulator I fixed the bullet point to say "Kill all processes" in a separate terminal.

* I created a new file called page121_Simulator_Testing_and_Evaluation_Notes.md:
    - Added my simulator test cases notes and added suggestions to improve the simulator.

Kait211 committed:
Update page120_working_with_jaia_software.md

Setting a Link to setting up user and configuring permissions

Kait211 committed:
Update page120_working_with_jaia_software.md

Steps to Setting up a Default New User

Kait211 committed:
Update page120_working_with_jaia_software.md

Kait211 committed:
Update page120_working_with_jaia_software.md

Finished Notes on setting up the simulator for default user

Kait211 committed:
Update page120_working_with_jaia_software.md

Adding a permission that is needed

Kait211 committed:
Update page120_working_with_jaia_software.md

Just some grammatical fixes

Kait211 committed:
Update page120_working_with_jaia_software.md

Just quick step/title fixes

Kait211 committed:
Update page120_working_with_jaia_software.md

One step grammar fix

Kait211 committed:
Update page120_working_with_jaia_software.md

Add another instruction for user about how you have to use a new terminal to kill all processes.

Kait211 committed:
Create page121_Simulator_Testing_and_Evaluation_Notes.md

Creating A new file in branch called page121_Simulator_Testing_and_Evaluation and adding a key.

Kait211 committed:
This will add table to the file I created

Creating Test Cases And Observations table

Kait211 committed:
Update page121_Simulator_Testing_and_Evaluation_Notes.md

Fixing table alignment

Kait211 committed:
Update page121_Simulator_Testing_and_Evaluation_Notes.md

Making sure Pictures are coming out correctly

Kait211 committed:
Update page121_Simulator_Testing_and_Evaluation_Notes.md

Finished the first column of test cases and observations table

Kait211 committed:
Update page121_Simulator_Testing_and_Evaluation_Notes.md

Fixing Table Alignment

Kait211 committed:
Update page121_Simulator_Testing_and_Evaluation_Notes.md

Completed Test Cases And Observations Table.

ferrom-jaia commented:
Hey Kaitlyn, 

Can you add a description to the pull request?

ferrom-jaia commented:
For the issues you ran into, are you able to post a short video of your experience?

Kait211 commented:
Yes, I can do that. I will begin working on it as soon as I am available.
________________________________
From: Matthew Ferro ***@***.***>
Sent: Tuesday, March 25, 2025 8:29 AM
To: jaiarobotics/jaiabot ***@***.***>
Cc: Kaitlyn Habib ***@***.***>; Author ***@***.***>
Subject: Re: [jaiarobotics/jaiabot] Task/update environment setup documentation (PR #1105)

You don't often get email from ***@***.*** Learn why this is important<https://aka.ms/LearnAboutSenderIdentification>
[EXTERNAL SENDER]

For the issues you ran into, are you able to post a short video of your experience?

—
Reply to this email directly, view it on GitHub<https://github.com/jaiarobotics/jaiabot/pull/1105#issuecomment-2751098693>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/A6CUXUT5BIRQZG35SJWIWGD2WFD3DAVCNFSM6AAAAABZC6NG32VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDONJRGA4TQNRZGM>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

[ferrom-jaia]ferrom-jaia left a comment (jaiarobotics/jaiabot#1105)<https://github.com/jaiarobotics/jaiabot/pull/1105#issuecomment-2751098693>

For the issues you ran into, are you able to post a short video of your experience?

—
Reply to this email directly, view it on GitHub<https://github.com/jaiarobotics/jaiabot/pull/1105#issuecomment-2751098693>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/A6CUXUT5BIRQZG35SJWIWGD2WFD3DAVCNFSM6AAAAABZC6NG32VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDONJRGA4TQNRZGM>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

Kait211 committed:
Update page121_Simulator_Testing_and_Evaluation_Notes.md

I am pushing the completed Suggestions for Improvement Table.

Kait211 committed:
Update page121_Simulator_Testing_and_Evaluation_Notes.md

Fixing link issues.

Kait211 committed:
Update page120_working_with_jaia_software.md

Just added a note and fix some comments.

Kait211 committed:
Update page120_working_with_jaia_software.md

Just fixing an alignment issue.

Kait211 committed:
Merge branch '1.y' into task/update-environment-setup-documentation

Kait211 committed:
Update page120_working_with_jaia_software.md

Fixed a grammar issue in a comment.

Kait211 committed:
Update page121_Simulator_Testing_and_Evaluation_Notes.md

Added a video on the issues I faced when using the simulator. Also added a new row to both my tables for a new issue I noticed when filming my video.

Kait211 committed:
Update page121_Simulator_Testing_and_Evaluation_Notes.md

Just added a better title and a description to my video.

Kait211 committed:
Update HelpWindow.tsx

Updating the help.

Kait211 committed:
Update HelpWindow.tsx

Just playing around with code to understand it

ferrom-jaia commented:
Kaitlyn and I talked about moving this work internally.

Great work!
